README.md

```diff
@@ -110,6 +110,17 @@ sudo apt-get install yank
 
 The binary is installed at `/usr/bin/yank-cli` due to a naming conflict.
 
+### Fedora
+
+Versions 24/25/26:
+
+```sh
+sudo dnf copr enable nmilosev/yank
+sudo dnf install yank
+```
+
+Binary is at `/usr/bin/yank-cli`, man-pages are both `yank` and `yank-cli`.
```

Comment:
Rephrase this sentence to mimic the Debian instructions:

> The binary is installed at `/usr/bin/yank-cli` due to a naming conflict,
> man-pages are available as both `yank` and `yank-cli`.

Make sure to amend or squash so we end up with a single commit.